src/pytest_reportlog/plugin.py

```diff
@@ -17,6 +17,13 @@ def pytest_addoption(parser):
     )
 
 
+@pytest.fixture(scope="function")
+def report_log(record_property):
```

Comment:
Oh sorry, I don't think I explained myself correctly: we should just include the contents of `item.user_properties` in the logreport, instead of creating a new fixture.

Reply:
Ah ok, I was wondering if I should let as ``junitxml`` or we should add those contents
I will fix it then, thanks

Reply:
> I was wondering if I should let as junitxml 

Hmm what do you mean?

Reply:
I updated the code, is it correct now? Sorry if I misunderstood that again :grimacing: 

* I think I expressed myself in the wrong way, sorry. What I meant was, I was in doubt if I should let the fixture or just unpack the values into the report, now (I think) I did as you requested

Reply:
Thanks!

Actually I think we should keep the current behavior of having `user_properties` as a standalone key in the report.

In the end I think we don't need to change anything and `record_property` is the fixture you wanted in the first place, but due to my brain fart I didn't even realized that. 😁

Reply:
Hahaha, no problem! I'm going to close it then. Thanks!